src/rgw/rgw_lib.h

```diff
@@ -25,35 +20,32 @@ namespace rgw {
   class RGWLibFrontend;
 
   class RGWLib : public DoutPrefixProvider {
-    RGWFrontendConfig* fec;
+    AppMain main;
```

Comment:
`RGWLib` does hold a reference on `boost::intrusive_ptr<CephContext> cct` below, but on `RGWLib`'s destruction `cct` will be destroyed before `AppMain` is. should be able to reorder the member variables so `cct` comes first

Reply:
ok, thanks, that helps

Reply:
ok.  I'm cleaning up from a messy merge around dan's changes, will update this also